var/spack/repos/builtin/packages/umpire/package.py

```diff
@@ -15,12 +15,14 @@ class Umpire(CachedCMakePackage, CudaPackage, ROCmPackage):
     """An application-focused API for memory management on NUMA & GPU
     architectures"""
 
-    homepage = 'https://github.com/LLNL/Umpire'
-    git      = 'https://github.com/LLNL/Umpire.git'
+    homepage = 'https://github.com/asarkar-parsys/Umpire/'
+    git      = 'https://github.com/asarkar-parsys/Umpire.git'
```

Comment:
I don't think this is intended for `develop`? In any case you can override `git` for a single version if you want to add a feature branch.